src/custom/components/swap/EthWethWrap/WrapCard.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import { CurrencyAmount, Currency } from '@uniswap/sdk'
+import styled from 'styled-components'
+import CurrencyLogo from 'components/CurrencyLogo'
+import { SHORT_PRECISION } from 'constants/index'
+
+const BalanceLabel = styled.p<{ background?: string }>`
+  display: flex;
+  justify-content: center;
+  margin: 0.5rem 0;
+  width: 100%;
+
+  border-radius: ${({ theme }) => theme.buttonPrimary.borderRadius};
+
+  background: ${({ background = 'initial' }) => background};
+
+  > span {
+    &:first-child {
+      margin-right: 0.2rem;
+    }
+  }
+`
+
+const WrapCardWrapper = styled.div`
+  ${({ theme }) => theme.flexColumnNoWrap}
+  align-items: center;
+  justify-content: center;
+  flex: 1;
+  padding: 0.5rem;
+`
+
+export const WrapCardContainer = styled.div`
+  position: relative;
+  ${({ theme }) => theme.flexRowNoWrap}
+  border: 0.15rem solid ${({ theme }) => theme.bg1};
+  border-radius: ${({ theme }) => theme.buttonPrimary.borderRadius};
+  margin: 0.7rem 0;
+  width: 100%;
+
+  > ${WrapCardWrapper} {
+    &:nth-of-type(even) {
+      background-color: ${({ theme }) => theme.bg1};
+    }
+
+    > ${BalanceLabel}:last-child {
+      margin: 0;
+    }
+  }
+
+  // arrow
+  > svg {
+    position: absolute;
+    left: calc(50% - 15px);
```

Comment:
Can also simply do:

```
    top: 0;
    bottom: 0;
    margin: auto;
    right: 0;
```

Reply:
that's 2 extra lines :joy: but i can see the benefit of not needing to know svg size i guess